core/variant/variant_utility.cpp

```diff
@@ -352,6 +352,22 @@ bool VariantUtilityFunctions::is_equal_approx(double x, double y) {
 	return Math::is_equal_approx(x, y);
 }
 
+bool VariantUtilityFunctions::is_less_or_equal_approx(double x, double y) {
```

Comment:
We tend to spell out our words in methods.

Reply:
But the other math functions use approx too..

Reply:
I would like to make consistent with approx while 4.x, and reword to approximate by 5.0